app/Http/Controllers/BeatmapDiscussionPostsController.php

```diff
@@ -26,7 +26,7 @@ class BeatmapDiscussionPostsController extends Controller
 {
     public function __construct()
     {
-        $this->middleware('auth', ['except' => 'index']);
+        $this->middleware('auth', ['except' => 'index', 'show']);
```

Comment:
there are missing braces there resulting in auth middleware still being used for `show` action :upside_down_face: